spyderlib/config.py

```diff
@@ -724,6 +724,9 @@ def is_ubuntu():
 CONF = UserConfig('spyder', defaults=DEFAULTS, load=True, version=CONF_VERSION,
                   subfolder=SUBFOLDER, backup=True, raw_mode=True)
 
+# Ensures that the config is present on spyder first run
+CONF.set('main', 'interface_language', load_lang_conf())

```

Comment:
Yes, I think it's fine :-) But I think it's better to move it `configdialog.py`. Well, if it's possible because there is where this setting is really needed.